fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -3616,10 +3620,21 @@ struct RestoreCompleteTaskFunc : RestoreTaskFuncBase {
 		restore.clearApplyMutationsKeys(tr);
 
 		wait(taskBucket->finish(tr, task));
+
 		if (unlockDB) {
 			wait(unlockDatabase(tr, restore.getUid()));
 		}
 
+		bool bgRestoreEnabled = wait(restore.isBlobGranuleRestore().getD(tr));
+		if (bgRestoreEnabled) {
+			// Cleanup history keys after restore
```

Comment:
no plan to support historical access after full restore.. so cleanup those keys for historical access

Reply:
filed a task to track the s3 file cleanup after restore